x/oracle/types/ballot.go

```diff
@@ -86,6 +86,11 @@ func (pb ExchangeRateBallot) StandardDeviation() (sdk.Dec, error) {
 	sum := sdk.ZeroDec()
 	for _, v := range pb {
 		deviation := v.ExchangeRate.Sub(median)
+		// check for overflow
+		err := checkMulOverflow(deviation, deviation)
+		if err != nil {
+			continue
```

Comment:
Pretty sure this is the more-correct thing to do - in terra's implementation the standard deviation becomes zero : 

```golang
	defer func() {
		if e := recover(); e != nil {
			standardDeviation = sdk.ZeroDec()
		}
	}()
```

Which forces the ballot rewarding logic to use `RewardBand` instead of `𝜎` ([spec here](https://github.com/umee-network/umee/blob/main/x/oracle/spec/01_concepts.md#reward-band))

This seems maybe unintended or at least confusing, so I think we should just throw it out, use the others to calculate `𝜎`, and then let the `𝜀 = max(𝜎, R/2)` logic run regularly

Reply:
Ehhh yeah I'm not sure how I feel about having to port over decimal logic from the SDK just to check for overflow bounds here. Maybe the panic recovery is the cleaner option along with your suggestion above?

Reply:
If we can use panic-recover for code cleanliness, but have the logic throw out the offending vote and keep the rest, that seems like it would minimize an attacker's power to manipulate reward behavior (like a single validator being able to stop `𝜎` calculation and force use of `RewardBand`).